plotly/tools.py

```diff
@@ -1443,6 +1449,398 @@ class FigureFactory(object):
     """
 
     @staticmethod
+    def _unlabel_rgb(colors):
+        """
+        This function takes a list of two 'rgb(a, b, c)' color strings and
```

Comment:
🐄 Python doc strings typically follow one of the following patterns:

## Single line docstring

```
"""I fit on one line!"""
```

## Multi-line docstring

```
"""
Short summary

A more detailed explanation following a blank line.

"""
```